### PR TITLE
AnimationClipLoader add clip event support

### PR DESCRIPTION
**Pull request by luzhuang**
### Please check if the PR fulfills these requirements

- [x] The commit message follows our [guidelines](https://github.com/oasis-engine/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### Other information:

### CODE REVIEW

File: packages/loader/src/gltf/extensions/GLTFExtensionSchema.ts [hunk 1]
```diff
@@ -159,14 +159,22 @@ export interface IOasisMaterialRemap {
...
+}
+
 export type GLTFExtensionSchema =
   | IKHRLightsPunctual_Light
   | IKHRDracoMeshCompression
   | IKHRMaterialsClearcoat
   | IKHRMaterialsIor
   | IKHRMaterialsUnlit
   | IKHRMaterialsPbrSpecularGlossiness
-  | IKHRMaterialsSheen
```
Comment (GuoLei1990) [inline packages/loader/src/gltf/extensions/GLTFExtensionSchema.ts:169]: delete `w`

File: packages/loader/src/gltf/extensions/OASIS_animation.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { AnimationClip, AnimationEvent } from "@oasis-engine/core";
+import { GLTFResource } from "../GLTFResource";
+import { GLTFExtensionOwnerSchema } from "../GLTFSchema";
+import { registerGLTFExtension } from "../parser/GLTFParser";
+import { GLTFParserContext } from "../parser/GLTFParserContext";
+import { GLTFExtensionMode, GLTFExtensionParser } from "./GLTFExtensionParser";
+import { IOasisAnimation } from "./GLTFExtensionSchema";
+
+// @ts-ignore
```
Comment (GuoLei1990) [inline packages/loader/src/gltf/extensions/OASIS_animation.ts:9]: delete  `@ts-ignore`

File: packages/loader/src/gltf/extensions/OASIS_animation.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+@registerGLTFExtension("OASIS_animation", GLTFExtensionMode.AdditiveParse)
```
Comment (GuoLei1990) [inline packages/loader/src/gltf/extensions/OASIS_animation.ts:10]: Maybe `OASIS_animationEvent`?

**Review by GuoLei1990 — APPROVED** [review]

File: packages/loader/src/gltf/extensions/OASIS_animation.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    const { engine } = context.glTFResource;
+    const { events } = schema;
+    return Promise.all(
+      events.map((eventData) => {
+        return new Promise<void>((resolve) => {
+          const event = new AnimationEvent();
+          event.functionName = eventData.functionName;
+          event.time = eventData.time;
+          event.parameter = eventData.parameter;
+          animationClip.addEvent(event);
```
Comment (zhuxudong) [inline packages/loader/src/gltf/extensions/OASIS_animation.ts:24]: why async

**Review by zhuxudong — APPROVED** [review]
+1

**Pull request merged**